video_xblock/video_xblock.py

```diff
@@ -388,18 +391,21 @@ def studio_view(self, _context):
         advanced_fields = self.prepare_studio_editor_fields(player.advanced_fields)
         log.debug("Fetched default transcripts: {}".format(self.default_transcripts))
         context = {
+            'advanced_fields': advanced_fields,
+            'auth_error_message': auth_error_message,
+            'basic_fields': basic_fields,
             'courseKey': self.course_key,
             'languages': languages,
+            'player_name': self.player_name,  # for players identification
+            'players': PlayerName,
+            'sources': TranscriptSource.to_dict().items(),
+            # transcripts context:
             'transcripts': transcripts,
```

Comment:
We need `transcripts` in the context - it's a kind of "all enabled transcripts" variable.